packages/imtr-client/src/index.js

```diff
@@ -93,12 +93,18 @@ export const getChecker = (config) => {
       const previousByUUID = question.uuid
         ? acc.find((q) => q.uuid === question.uuid)
         : null;
+
       if (previousByUUID) {
+        if (question.prio < previousByUUID.prio) {
+          // If new question has lower prio (more important) to override the value in our accumulator
+          previousByUUID.prio = question.prio;
+        }
```

Comment:
Always preserve lowest (most important) prio from duplicate questions. We want the first to make most sense.